internal/generator/fluentd/pipeline_to_output_test.go

```diff
@@ -102,9 +102,10 @@ var _ = Describe("Testing Config Generation", func() {
 <label @APP_TO_ES>
   # Add User Defined labels to the output record
   <filter **>
-    @type record_transformer
+    @type record_modifier
+    remove_keys _dummy_
     <record>
-      openshift { "labels": {"a":"b","c":"d"} }
+      _dummy_ ${record['openshift']={"labels"=>{}} unless record['openshift'];record['openshift']['labels'] = {"a"=>"b","c"=>"d"} }
```

Comment:
Why the _dummy_ field?

Reply:
Its the recommendation to basically "script" functionality without writing a plugin... and then the field gets removed

Reply:
/lgtm